javascript/budget-tool.js

```diff
@@ -228,6 +227,11 @@ window.wcb = window.wcb || {models:{}, input:[]};
             this.$el.toggleClass('has-changed', this.model.hasChanged() && ! this.model.get('new'));
             this.$el.toggleClass('is-new', this.model.get('new'));
             this.$el.data('wcb-cid', this.model.cid);
+
+			if( data.name === 'currency' && $.fn.hasOwnProperty( 'select2' ) ){
+				this.$el.find( 'select' ).select2( { width: '100%' } );
+			}
+
```

Comment:
Looks like there is an extra tab on the above lines.

Reply:
yeah, this is because the above is using spaces, and the new code uses tabs, but both of these have 3 levels of indentation. perhaps I can have a new pr with all code using tabs.